common/src/main/java/tds/common/configuration/CacheConfiguration.java

```diff
@@ -11,21 +12,19 @@
 import org.springframework.cache.interceptor.KeyGenerator;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-import org.springframework.context.annotation.Profile;
 import org.springframework.context.annotation.PropertySource;
-
-import java.util.concurrent.TimeUnit;
-
 import tds.common.cache.CacheKeyGenerator;
 import tds.common.cache.CacheType;
 
+import java.util.concurrent.TimeUnit;
+
 /**
  * Spring Configuration file for caching
  */
 @Configuration
 @EnableCaching
-@Profile("cache-enabled")
 @PropertySource(value="classpath:cache.properties", ignoreResourceNotFound=true)
+@ConditionalOnProperty(value = "tds.cache.enabled", havingValue = "true", matchIfMissing = false)
```

Comment:
Right now, caching is disabled by default.  It can be enabled by defining:
tds.cache.enabled = true
Since we want it disabled for most deployments in most environments during development, this made sense to me, plus it matches what was there.
If/when we deliver, we can change the matchIfMissing value to true to have it enabled by default.